README.md

```diff
@@ -1 +1,32 @@
-# To-Do-List
\ No newline at end of file
+# To Do List Project
+
+> This is simple to do list project which is under building. For now there is no functionlity to use.
+
+
+## Built With
```

Comment:
- [ ] your readme file should have a section that guides your visitors on how to Setup&Install the project, kindly add the necessary instructions to make your readme more professional 🙏